x-pack/platform/plugins/shared/spaces/public/management/components/customize_space/customize_space.test.tsx

```diff
@@ -78,7 +78,7 @@ test('updates identifier, initials and color when name is changed', () => {
     id: 'space-2',
     name: 'Space 2',
     initials: 'S2',
-    color: '#9170B8',
+    color: '#BFDBFF',
```

Comment:
Nit: it would be nice if these color values were constants somewhere that could be referenced across all tests.

Reply:
100% agree. It's on our todo list and I'm planning to address this in coming weeks